css/logo-nav.css

```diff
@@ -628,70 +627,56 @@ a {
     margin-bottom: 10px;
     cursor: default;
 } 
+
 .clicky:hover {
     color: #E46E2E
 }
-.institutional .items {
+
+.items {
     border-color: black;
     border-width: thin;
     border-style: solid;
     border-radius: 5px;
     border-color: #979797;
-    height: 100px;
+    padding: 30px;
+    transition: all 1s;
+    min-height: 100px;
 }
 .content-gov {
-    margin-top: 18px;
     padding-left: 20px;
     padding-right: 35px;
     color: #818181;
 }
-.institutional .menu {
-    height: 30%
-}
+
+
 #institutional {
     background-color: #F9F9F9;
-
-}
-.abs {
-  position: absolute;
-  top: 50px;
 }
+
 .menuClicked {
     color: #E46E2E;
 }
-@media (max-width: 687px) {
-    .content-gov {
-        margin-top: 60px;
-        padding-bottom: 60px;
-        vertical-align: center;
+
+@media (max-width: 509px) {
+    .items {
+        padding:10px;
+        min-height:0px;
     }
 
-}
-@media (max-width: 599px) {
     .content-gov {
-        margin-top: 50px;
-    }
-}
-@media (max-width: 509px) {
-    .institutional .items {
-        height: 120px;
+        margin: 0px;
+        padding: 10px;
     }
 }
 @media (max-width: 395px) {
     .content-gov {
         margin-top: 90px;
     }
-    .institutional .items {
-        height: 140px;
-    }
 }
 @media (max-width: 365px) {
     .content-gov {
         margin-top: 130px;
     }
-    .institutional .items {
-        height: 160px;
-    }
```

Comment:
All these manual setting of height were unnecessary once `.abs` was removed and the height of wells calculated by the browser from contents height.